pkg/proxy/winkernel/hnsV1.go

```diff
@@ -75,6 +75,10 @@ func (hns hnsV1) getEndpointByIpAddress(ip string, networkName string) (*endpoin
 	}
 
 	endpoints, err := hcsshim.HNSListEndpointRequest()
+	if err != nil {
+		klog.Errorf("%v", err)
```

Comment:
I think that for v1 (and the v2 code below) we may want use the error included in the return statement? I also think we can skip the `klog.Errorf`... typically its the responsibility of whatever calls the func to handle the error.

Reply:
Was following the existing pattern on line 73 of the same file; but will fix!